React/lottery-react/README.md

```diff
@@ -0,0 +1,24 @@
+# Lottery React
+
+## Description
+Connect with other people on a rinkeby network. Bet some eth tokens and enter the lottery.
+
+## Tech Stack
+web3, React
+
+## Show Steps on how to run in Local
+
+In `/web`
+Edit the .env.example to .env and provide all the data    
+Then
+```bash
+npm i
+node deploy.js
+npm test # should pass all test
+```
+
+In root
+```
+npm i
+npm start
+```
```

Comment:
Visit #178 and add demo video.

Reply:
Yes I have added a working video Please look into it